brayns/common/types.h

```diff
@@ -308,7 +312,8 @@ enum class CameraType
     perspective,
     stereo,
     orthographic,
-    panoramic
+    panoramic,
+    clipped
```

Comment:
Why should the clipping be attached to the scene? We are not selecting the data there, only visualizing a subset of it. The scene should definitely not be affected by the clipping planes. The camera definitely is the place where the clip planes should be handled.